plugins/logging/logging.cpp

```diff
@@ -125,41 +125,25 @@ LOGGING_API unsigned int LOGGING_CALL logGetElapsedMs(ICodeContext *ctx)
 
 LOGGING_API char * LOGGING_CALL getTraceID(ICodeContext *ctx)
 {
-    StringBuffer ret;
-    Owned<IProperties> httpHeaders = ctx->queryContextLogger().getSpanContext();
-    if (httpHeaders)
-        ret.set(httpHeaders->queryProp("traceID"));
-
+    StringBuffer ret(ctx->queryContextLogger().getSpanContext()->queryProp("traceID"));
```

Comment:
This change isn't correct - it will now leak the properties (we return pointers rather than smart pointers).